### PR TITLE
More Roadmap updates

### PR DESCRIPTION
**Pull request by paulb777**
Updates after today's discussion

#no-changelog

### CODE REVIEW

**Review by maksymmalyhin — APPROVED** [review]
LGTM👍

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ template_path=.github/ISSUE_TEMPLATE/feature_request.md
 -->
 ### [READ] Guidelines
 
+Consider instead starting a
+[Pitch](https://github.com/firebase/firebase-ios-sdk/discussions/categories/pitches)
+discussion for ideas that you'd like to discuss with the Firebase community to
+flush out.
+
```
Comment (morganchen12) [inline .github/ISSUE_TEMPLATE/feature_request.md:16]: We should add some criteria here that disambiguates whether or not users should file a pitch or feature request. Maybe something like "For large or ambiguous requests, such as significant breaking changes or use cases that could be addressed by multiple different features, consider ..."

**Review by morganchen12 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (paulb777) [reply .github/ISSUE_TEMPLATE/feature_request.md:18]: Done Thanks.

**Comment by paulb777** [comment]
Merging now and can address any subsequent comments in another PR

**Pull request merged**